src/screens/DetailDomain/elements/DetailDomainScreenHeader.js

```diff
@@ -137,7 +137,7 @@ const styles = StyleSheet.create({
     paddingRight: 20
   },
   followerNumber: {
-    color: COLORS.blackgrey,
+    color: '#828282',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, avoid using hardcoded color values. Use the `COLORS` object for color definitions.
```diff
-    color: '#828282',
+    color: COLORS.blackgrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->